src/Microsoft.OpenApi/Models/OpenApiContact.cs

```diff
@@ -45,10 +45,10 @@ public class OpenApiContact : IOpenApiSerializable, IOpenApiExtensible
         /// </summary>
         public OpenApiContact(OpenApiContact contact)
         {
-            Name = contact.Name;
-            Url = new Uri(contact.Url.OriginalString);
-            Email = contact.Email;
-            Extensions = new Dictionary<string, IOpenApiExtension>(contact.Extensions);
+            Name = contact?.Name;
```

Comment:
```suggestion
            Name = contact?.Name ?? Name;
```

And repeat for scalar values in case there are defaults